gui/_window.scss

```
@@ -69,7 +69,7 @@
   &-footer,
   footer {
     margin: var(--window-spacing);
-    margin-top: calc(-var(--window-spacing) - 1px);
+    margin-top: calc(0 - var(--window-spacing) - 1px);
```

Hey mate, as in the stackoverflow answer you linked, it seems that `0 - var(--window-spacing)` might not work as we would expect, hence the change I suggest below.

```suggestion
    margin-top: calc(-1 * var(--window-spacing) - 1px);
```